Decred Wallet/Features/Settings/Settings.storyboard

```diff
@@ -503,7 +503,7 @@
                                         </connections>
                                     </tableViewCell>
                                     <tableViewCell clipsSubviews="YES" contentMode="scaleToFill" preservesSuperviewLayoutMargins="YES" selectionStyle="default" indentationWidth="10" reuseIdentifier="delete_wallet_settings" textLabel="dgz-JW-F6y" style="IBUITableViewCellStyleDefault" id="JNZ-nH-m85">
-                                        <rect key="frame" x="0.0" y="1041.5" width="375" height="44"/>
+                                        <rect key="frame" x="0.0" y="927.5" width="375" height="44"/>
                                         <autoresizingMask key="autoresizingMask"/>
                                         <tableViewCellContentView key="contentView" opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" preservesSuperviewLayoutMargins="YES" insetsLayoutMarginsFromSafeArea="NO" tableViewCell="JNZ-nH-m85" id="ug0-e8-luS">
                                             <rect key="frame" x="0.0" y="0.0" width="375" height="44"/>
```

Comment:
i don't think this file should be touched in this PR...or was it referenced?

Reply:
It is a postfix to PR #548. Fix this padding bug: 
<img width="496" alt="Screenshot 2020-01-07 at 17 00 18" src="https://user-images.githubusercontent.com/52497040/71909384-3b58e080-3170-11ea-9bff-0d3fec3ead13.png">

Reply:
The new UI is implemented in #559
<img width="425" alt="Screenshot 2020-01-07 at 17 29 40" src="https://user-images.githubusercontent.com/794430/71911043-4cefb780-3173-11ea-92c0-5764221491df.png">

Reply:
Ok, I have reverted it.